d2common/d2data/d2datadict/hireling.go

```diff
@@ -109,7 +109,7 @@ func LoadHireling(file []byte) {
 			HP:              d.Number("HP"),
 			HPPerLvl:        d.Number("HP/Lvl"),
 			Defense:         d.Number("Defense"),
-			DefPerLvl:       d.Number("Id"),
+			DefPerLvl:       d.Number("ID"),
```

Comment:
`ID`?

Reply:
@Iniquitatis is correct, the .txt file column is `Id` and not `ID`

Reply:
Ah, no, I'm talking about the parameter `DefPerLvl`, which should be read from appropriate table field, but not `ID`, I think?
Still, I might be wrong.

Reply:
ah, there's two errors here. I tunnel-visioned on the column string used for field retrieval, but you are totally correct in regard to where this value is being assigned.